.circleci/config.yml

```diff
@@ -34,47 +34,40 @@ jobs:
       - run:
           name: Write the sha256 sum to an artifact for verification
           command: |
-            docker images --no-trunc | awk '/^app/ {print $3}' | tee $CIRCLE_ARTIFACTS/docker-image-shasum256.txt
+            docker images --no-trunc | awk '/^app/ {print $3}' | sudo tee $CIRCLE_ARTIFACTS/docker-image-shasum256.txt
       - run:
           name: Clean up any old images and save the new one.
           command: |
-            I="image-$(date +%j).gz"; mkdir -p ~/docker; rm ~/docker/*
-            docker save addons-frontend | pigz --fast -c > ~/docker/$I
-            ls -l ~/docker
-      - save_cache:
-          key: docker-cache-{{ checksum "~/docker" }}
-          paths:
-            - ~/docker
+            docker save addons-frontend | pigz --fast -c > ~/addons-frontend/docker/$I
+            ls -l ~/addons-frontend/docker
       - persist_to_workspace:
           root: .
           paths:
             - ./*
   release-tag: # build for releases (tags)
     machine: true
-    working_directory: ~/addons-frontend
     steps:
       - attach_workspace:
           at: .
       - run:
           name: Build docker image and push to repo
           command: |
             [ ! -z $DOCKERHUB_REPO ]
-            docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Please also remove `DOCKER_EMAIL` from the comment at the top of the file. It looks like it's no longer needed.